### PR TITLE
[ADD] Custom toast, for the everything on the WEB

### PR DESCRIPTION
**Pull request by H-chauvet**
Custom toast is displayed in every pages

The toast has a title "Success" or "Error" based on the context

Green and red color based on the artistic direction are used for the custom toast

Icon "Check" or Icon "Cross" is used depending if it's a success or an error

Progress bar showing the remaining time before the toast disappears

Capture d’écran du 2024-07-17 20-48-13
Capture d’écran du 2024-07-17 20-47-58

### CODE REVIEW

File: Configurateur/front/lib/network/informations.dart
Comment (Louismaestre) [inline Configurateur/front/lib/network/informations.dart:1]: I think you push a wrong file

Comment (H-chauvet) [reply Configurateur/front/lib/network/informations.dart:1]: Fixed

**Review by Louismaestre — APPROVED** [review]

File: Configurateur/front/lib/components/tickets_page.dart [hunk 1]
```diff
@@ -114,12 +111,7 @@ class TicketsState extends State<TicketsPage> {
     );
     if (response.statusCode == 201) {
     } else {
-      Fluttertoast.showToast(
-          msg: "Erreur durant la cloture du ticket",
-          toastLength: Toast.LENGTH_LONG,
-          gravity: ToastGravity.CENTER,
-          timeInSecForIosWeb: 3,
-          backgroundColor: Colors.red);
+      showCustomToast(context, "Erreur durant la cloture du ticket", false);
```
[the remark this answers is not in the document]
Comment (H-chauvet) [reply Configurateur/front/lib/components/tickets_page.dart]: done

File: Configurateur/front/lib/screens/company-profil/company-profil.dart [hunk 2]
```diff
@@ -142,21 +135,11 @@ class CompanyProfilPageState extends State<CompanyProfilPage> {
     );
 
     if (response.statusCode == 200) {
-      Fluttertoast.showToast(
-        msg: 'Modification effectuée avec succès',
-        toastLength: Toast.LENGTH_LONG,
-        gravity: ToastGravity.CENTER,
-        timeInSecForIosWeb: 3,
-      );
+      showCustomToast(context, "Modification effectuée avev succès !", true);
```
[the remark this answers is not in the document]
Comment (H-chauvet) [reply Configurateur/front/lib/screens/company-profil/company-profil.dart]: done

**Review by hugolopespinto — APPROVED** [review]

**Pull request merged**